web.go

```diff
@@ -60,7 +60,12 @@ func runWeb(mux *http.ServeMux, logger log.Logger, db storage.Queryable, reloadC
 	api := api.New(log.With(logger, "component", "pprofui"), db, reloadCh)
 
 	router.GET("/-/reload", api.Reload)
+
+	// TODO(bwplotka): Consider replacing this with remote read https://github.com/conprof/conprof/issues/80.
```

Comment:
As I said in the issue I think this needs more discussion. Let's remove this comment for now.

Reply:
agree